x-pack/plugins/observability_solution/slo/server/lib/rules/slo_burn_rate/lib/should_suppress_instance_id.test.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SLO } from '../../../../domain/models';
+import { Rule } from '@kbn/alerting-plugin/common';
+import { BurnRateRuleParams } from '../types';
+import { shouldSuppressInstanceId } from './should_suppress_instance_id';
+import { ALL_VALUE } from '@kbn/slo-schema';
+
+describe('shouldSuppressInstanceId', () => {
```

Comment:
👍🏻 These tests are very useful to understand the logic and act as live documentation